R-package/R/lightgbm.R

```diff
@@ -73,24 +73,31 @@ lightgbm <- function(data,
     dtrain <- lgb.Dataset(data, label = label, weight = weight)
   }
 
+  train_args <- list(
+    "params" = params
+    , "data" = dtrain
+    , "nrounds" = nrounds
+    , "verbose" = verbose
+    , "eval_freq" = eval_freq
+    , "early_stopping_rounds" = early_stopping_rounds
+    , "init_model" = init_model
+    , "callbacks" = callbacks
```

Comment:
is `...` missed?

Reply:
ah, yes! Good catch. I'll add that and a test on it

Reply:
oh wait @guolinke no I have it!

```
train_args <- append(train_args, list(...))
```

Reply:
and it's being tested already because in my tests I have `metric = "binary_error"` (and `metric`  is not a keyword argument of the `lightgbm()` function)